src/postgres/tree/PostgresResourcesTreeItemBase.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ClientConfig } from "pg";
+import { AzureParentTreeItem, ISubscriptionContext } from "vscode-azureextensionui";
+import { PostgresDatabaseTreeItem } from "./PostgresDatabaseTreeItem";
+
+export abstract class PostgresResourcesTreeItemBase extends AzureParentTreeItem<ISubscriptionContext> {
```

Comment:
`PostgresResourcesTreeItemBase` is a pretty generic name, but I don't have a better name for you. Maybe just add a comment in the file describing when and how this is meant to be used